hive_metastore_client/hive_metastore_client.py

```diff
@@ -156,6 +164,35 @@ def add_partitions_if_not_exists(
         except AlreadyExistsException:
             pass
 
+    def add_partitions_to_table(
```

Comment:
since we're removing the `AlreadyExistsException`, there won't be a scenario where I can trying adding the same values again and not receive an error?

Reply:
According to what we already talked, I updated this PR keeping the method `add_partitions_if_not_exists`, but fixing its behavior.
Also, I kept the new `add_partitions_to_table`.